HHS/Head-Start-TTADP
# Initial backend

A very basic initial backend with docker and nodemon.

To test it:

`npm run docker:start`
go to localhost:3000 - server should output a test message
modify src/index.js to change the message
reload localhost:3000 - see the changed message
